docker-compose.yaml

```diff
@@ -1,12 +1,37 @@
-version: '3.9'
-
 services:
-
-  db:
+  database:
     image: postgres:14.2
-    restart: always
+    mem_limit: 2g
     environment:
       POSTGRES_DB: argo
       POSTGRES_PASSWORD: example
+      PGDATA: /var/lib/postgresql/data/pgdata
+    volumes:
+      - ./db/data:/var/lib/postgresql/data
     ports:
-      - "5432:5432"
+      - 5432:5432
+  backend:
+    profiles: ["backend", "complete"]
+    image: mwalbeck/python-poetry:latest
```

Comment:
I prefer not to use "latest" tag anywhere. Let's lock it to some specific version.

Reply:
changed to 1.1.13-3.10